{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -147,6 +150,32 @@ search = "\"version\": \"{current_version}\""
 replace = "\"version\": \"{new_version}\""
 {%- endif %}
 
+
```

Comment:
@bzah You just need to remove this line and I think the tests will pass!

Reply:
oops, that's what I get for not running the tests locally 😅